### DISCUSSION TITLE
Random Number Generator 3.0.0

### DISCUSSION THREAD

**Discussion by 12sdj** [12sdj/Random-Number-Generator#9] [Announcements]
Welcome to Random Number Generator, we insist on the concept of 'always new', to achieve perfection in software usability and appearance as much as possible.
[Changelog]
Release Procedure

<hr /><em>This discussion was created from the release <a href='https://github.com/12sdj/Random-Number-Generator/releases/tag/3.0.0'>Random Number Generator 3.0.0</a>.</em>

**Comment by YKJ12sdj** [comment]
wow！That's good!
